src/Storefront/Resources/snippet/en_GB/storefront.en-GB.json

```diff
@@ -733,7 +733,7 @@
     "basicCaptchaLabel": "To continue, enter the characters shown above*"
   },
   "cookie": {
-    "messageText": "This website uses cookies to ensure the best experience possible. <a data-ajax-modal=\"true\" data-url=\"%url%\" href=\"%url%\" title=\"More information\">More information...</a>",
+    "messageText": "This website uses cookies to ensure the best experience possible. <button class=\"btn btn-link-inline\" data-ajax-modal=\"true\" data-url=\"%url%\">More information...</button>",
```

Comment:
German change missing

Reply:
Change was reverted because here we will use the new snippet with link.